setup.cfg

```diff
@@ -1,4 +1,13 @@
+[metadata]
+license_file = COPYING
+
+[aliases]
+test=pytest
+
 [flake8]
 ignore = E122,E501,H101,H105,H201,H301,H404,H405,B001,W504
 import-order-style = google
 application-import-names = __isocodes__,__version__,AbstractCheck,AppDataCheck,BinariesCheck,Config,ConfigCheck,DistributionCheck,DocFilesCheck,FHSCheck,FilesCheck,Filter,I18NCheck,InitScriptCheck,LSBCheck,MenuCheck,MenuXDGCheck,NamingPolicyCheck,PamCheck,Pkg,PostCheck,RpmFileCheck,SCLCheck,SignatureCheck,SourceCheck,SpecCheck,TagsCheck,ZipCheck,Testing
+
+[tool:pytest]
+addopts = --flake8 --cov=rpmlint
```

Comment:
can we run flake8 separately, via tox perhaps? having flake8 as a test dependency is inconvenient for building the package in a distribution (pulls in all the flake dependencies).

Reply:
The distribution can still override the pytest and disable the flake8 run if they desire so. Yet it should not cause issues for us, and this way we don't need no tox to be present in the enviroment.